nsqd/http.go

```diff
@@ -131,6 +131,14 @@ func (s *httpServer) doInfo(w http.ResponseWriter, req *http.Request, ps httprou
 	if err != nil {
 		return nil, http_api.Err{500, err.Error()}
 	}
+	tcpAddr, ok := s.nsqd.RealTCPAddr().(*net.TCPAddr)
+	if !ok {
+		return nil, http_api.Err{500, "failed to cast to TCPAddr"}
+	}
+	httpAddr, ok := s.nsqd.RealHTTPAddr().(*net.TCPAddr)
+	if !ok {
+		return nil, http_api.Err{500, "failed to cast to TCPAddr"}
```

Comment:
does this mean that the nsqd info endpoint fails if unix socket is used? we can use port `-1` or similar for unix sockets

Reply:
Done